fdbserver/Status.actor.cpp

```diff
@@ -2153,13 +2153,25 @@ ACTOR Future<JsonBuilderObject> lockedStatusFetcher(Reference<AsyncVar<CachedSer
 ACTOR Future<Optional<Value>> getActivePrimaryDC(Database cx) {
 	state ReadYourWritesTransaction tr(cx);
 
+	state Future<Void> readTimeout = delay(10); // so that we won't loop forever
```

Comment:
I think the timeouts in here are kept a little lower, maybe 2-5 seconds.
```suggestion
	state Future<Void> readTimeout = delay(5); // so that we won't loop forever
```